client/src/tutorials/Tic-tac-toe/files/index.test.ts.raw

```diff
@@ -0,0 +1,69 @@
+
+describe("Tictactoe game", async () => {
+  const systemProgram = anchor.web3.SystemProgram;
+  const player1 = pg.wallet.publicKey;
+  const player2 = pg.wallet.publicKey;
```

Comment:
FYI, we can use multiple wallets at the same time here:

```
pg.wallets.<walletName>
```

(Add account from wallet settings)

Reply:
Good to know. Keep the same for now i feel.